aequilibrae/paths/linear_approximation.py

```diff
@@ -426,6 +435,11 @@ def execute(self):
                 self.cores,
             )
 
+            for c in self.traffic_classes:
+                if self.time_field in c.graph.skim_fields:
+                    k = c.graph.skim_fields.index(self.time_field)
+                    aggregate_link_costs(self.congested_time[:], c.graph.compact_skims[:, k], c.results.crosswalk)
```

Comment:
This is the line where we put the congested time back into the skim array, if that's being skimmed

![image](https://user-images.githubusercontent.com/7377666/160235797-b95186e2-9e11-4b42-bd08-d069df886946.png)